tests_recording/test_status.py

```diff
@@ -20,24 +20,77 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import unittest
 
-import pkg_resources
+from requre.cassette import DataTypes
+from requre.online_replacing import (
+    apply_decorator_to_all_methods,
+    record_requests_for_all_methods,
+    replace_module_match,
+)
+from requre.helpers.tempfile import TempFile
+from requre.helpers.simple_object import Simple
+from requre.helpers.files import StoreFiles
+from requre.helpers.git.pushinfo import PushInfoStorageList
+from requre.helpers.git.repo import Repo
 
 from packit.status import Status
-from requre.cassette import DataTypes
-from tests_recording.testbase import PackitUnittestOgr
+from tests_recording.testbase import PackitTest
 
 
-@unittest.skip("Not working yet")
-class TestStatus(PackitUnittestOgr):
+@record_requests_for_all_methods()
+@apply_decorator_to_all_methods(
+    replace_module_match(
+        what="packit.utils.run_command_remote", decorate=Simple.decorator_plain()
+    )
+)
+@apply_decorator_to_all_methods(
+    replace_module_match(
+        what="packit.fedpkg.FedPKG.clone",
+        decorate=StoreFiles.where_arg_references(
+            key_position_params_dict={"target_path": 2}
+        ),
+    )
+)
```

Comment:
It would be nice to somehow share those definitions if they are same.

Solution here:
- Will the decorating of the super class work?
- Have a new decorator that will inlcude all of these. (Really easy.)

Improvements on requre side:
- Be able to apply mutliple replacements to the `apply_decorator_to_all_methods`.

Reply:
I have also idea with superclass, but it is applied to class, so method are decorated just in superclass, not in all of them.
Yes, solution with decorator what will cover all others, is simple and possible, but my feeling is that this is more explicit so maybe easier to understand, what exactly is applied, but yep, I'll do that.